Makefile

```diff
@@ -155,7 +164,7 @@ bandit: requirements .bandit
 lint: requirements .lint
 
 .PHONY: .lint
-.lint: .flake8 .pylint .bandit .st2client-dependencies-check .st2common-circular-dependencies-check .rst-check
+.lint: .generate-openapi-spec .flake8 .pylint .bandit .st2client-dependencies-check .st2common-circular-dependencies-check .rst-check
```

Comment:
tbh, I also didn't know where exactly to put this target.

It needs to be somewhere so it's executed on every make run so it always gets generated and committed.

If it's skipped and people don't commit it, things will get out of sync and it will cause issues.